go/filemanager/pfsmod/chunk.go

```diff
@@ -0,0 +1,118 @@
+package pfsmod
+
+import (
+	"errors"
+	"fmt"
+	log "github.com/golang/glog"
+	"io"
+	"net/url"
+	"os"
+	"strconv"
+)
+
+//ChunkCmd structure
+type ChunkCmd struct {
+	Path      string
+	Offset    int64
+	ChunkSize int64
+	Data      []byte
+}
+
+//NewChunkCmd get a new ChunkCmd
+func NewChunkCmd(path string, offset, chunkSize int64) *ChunkCmd {
+	return &ChunkCmd{
+		Path:      path,
+		Offset:    offset,
+		ChunkSize: chunkSize,
+	}
+}
+
+//ToURLParam encodes variables to url encoding parameters
+func (p *ChunkCmd) ToURLParam() string {
+	parameters := url.Values{}
+	parameters.Add("path", p.Path)
+
+	str := fmt.Sprint(p.Offset)
+	parameters.Add("offset", str)
+
+	str = fmt.Sprint(p.ChunkSize)
+	parameters.Add("chunksize", str)
+
+	return parameters.Encode()
+}
+
+//ToJSON encodes chunkcmd to json string
+func (p *ChunkCmd) ToJSON() ([]byte, error) {
+	return nil, nil
+}
+
+//Run function runs a ChunkCmd
+func (p *ChunkCmd) Run() (interface{}, error) {
+	return nil, nil
```

Comment:
Why it's returning `nil, nil`, is it not implemented? If so, do you plan to implement it later? If so, maybe `panic("not implemented")` is better.